config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.graph

```diff
@@ -0,0 +1,3 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
```

Comment:
I just tested and it seems that comments are not allowed in .graph files, was the migration successful for you ?

Reply:
Ah I didn't know that, sorry, I've removed the comment, it should work for you.

Besluit over budget(wijziging) eredienstbestuur
![Screenshot 2023-02-22 at 09-21-38 Loket voor lokale besturen](https://user-images.githubusercontent.com/38471754/220564312-c7b70697-20db-4f36-9c00-17de9b3901ac.png)

Besluit over meerjarenplan(aanpassing) eredienstbestuur
![Screenshot 2023-02-22 at 09-22-04 Loket voor lokale besturen](https://user-images.githubusercontent.com/38471754/220564553-715318fc-d87e-43ad-9f00-c4051dbbd43f.png)